subiquity/ui/views/filesystem/guided.py

```diff
@@ -155,12 +155,12 @@ class TPMChoice:
             enabled=False, default=False,
             help=_("The model being installed prefers but does not require "
                    "TPM backed full-disk encryption and it is not available "
-                   "on this device")),
+                   "on this device (the reason given was \"{reason}\")")),
```

Comment:
minor: Full stop for consistency with other similar messages?

Reply:
:+1: